src/constants/icons.ts

```diff
@@ -0,0 +1,103 @@
+import CHEVRON_RIGHT_16 from '@assets/icons/chevron_right_16.svg'
+import CHEVRON_RIGHT_40 from '@assets/icons/chevron_right_40.svg'
+import CLOCK_16 from '@assets/icons/clock_16.svg'
+import CLOCK_24 from '@assets/icons/clock_24.svg'
+import CLOSE_16 from '@assets/icons/close_16.svg'
+import CLOSE_24 from '@assets/icons/close_24.svg'
+import FILTER_16 from '@assets/icons/filter_16.svg'
+import FILTER_24 from '@assets/icons/filter_24.svg'
+import GOOGLE_24 from '@assets/icons/google_24.svg'
+import HEART_16 from '@assets/icons/heart_16.svg'
+import HEART_24 from '@assets/icons/heart_24.svg'
+import HEART_ACTIVE_16 from '@assets/icons/heart_active_16.svg'
+import HEART_ACTIVE_24 from '@assets/icons/heart_active_24.svg'
+import KAKAO_24 from '@assets/icons/kakao_24.svg'
+import MEH_40 from '@assets/icons/meh_40.svg'
+import MEH_ACTIVE_24 from '@assets/icons/meh_active_24.svg'
+import MEH_ACTIVE_40 from '@assets/icons/meh_active_40.svg'
+import MENU_24 from '@assets/icons/menu_24.svg'
+import MESSAGE_100 from '@assets/icons/message_100.svg'
+import MESSAGE_16 from '@assets/icons/message_16.svg'
+import MESSAGE_24 from '@assets/icons/message_24.svg'
+import MORE_16 from '@assets/icons/more_16.svg'
+import MORE_24 from '@assets/icons/more_24.svg'
+import PICTURE_40 from '@assets/icons/picture_40.svg'
+import REFRESH_16 from '@assets/icons/refresh_16.svg'
+import REFRESH_24 from '@assets/icons/refresh_24.svg'
+import SAD_40 from '@assets/icons/sad_40.svg'
+import SAD_ACTIVE_24 from '@assets/icons/sad_active_24.svg'
+import SAD_ACTIVE_40 from '@assets/icons/sad_active_40.svg'
+import SEARCH_24 from '@assets/icons/search_24.svg'
+import SETTING_24 from '@assets/icons/setting_24.svg'
+import SMILE_16 from '@assets/icons/smile_16.svg'
+import SMILE_24 from '@assets/icons/smile_24.svg'
+import SMILE_40 from '@assets/icons/smile_40.svg'
+import SMILE_ACTIVE_24 from '@assets/icons/smile_active_24.svg'
+import SMILE_ACTIVE_40 from '@assets/icons/smile_active_40.svg'
+import STORE_16 from '@assets/icons/store_16.svg'
+import STORE_24 from '@assets/icons/store_24.svg'
+import TRIANGLE_DOWN_16 from '@assets/icons/triangle_down_16.svg'
+
+export const ICON = {
+  ARROW_LEFT_24,
+  ARROW_UP_24,
+  BOX_16,
+  CAMERA_16,
+  CAMERA_24,
+  CHECK_CIRCLE_16,
+  CHECK_CIRCLE_FILL_100,
+  CHEVRON_DOWN_16,
+  CHEVRON_LEFT_16,
+  CHEVRON_LEFT_40,
+  CHEVRON_RIGHT_14,
+  CHEVRON_RIGHT_16,
+  CHEVRON_RIGHT_40,
+  CLOCK_16,
+  CLOCK_24,
+  CLOSE_16,
+  CLOSE_24,
+  FILTER_16,
+  FILTER_24,
+  GOOGLE_24,
+  HEART_16,
+  HEART_24,
+  HEART_ACTIVE_16,
+  HEART_ACTIVE_24,
+  KAKAO_24,
+  MEH_40,
+  MEH_ACTIVE_24,
+  MEH_ACTIVE_40,
+  MENU_24,
+  MESSAGE_100,
+  MESSAGE_16,
+  MESSAGE_24,
+  MORE_16,
+  MORE_24,
+  PICTURE_40,
+  REFRESH_16,
+  REFRESH_24,
+  SAD_40,
+  SAD_ACTIVE_24,
+  SAD_ACTIVE_40,
+  SEARCH_24,
+  SETTING_24,
+  SMILE_16,
+  SMILE_24,
+  SMILE_40,
+  SMILE_ACTIVE_24,
+  SMILE_ACTIVE_40,
+  STORE_16,
+  STORE_24,
+  TRIANGLE_DOWN_16
```

Comment:
```suggestion
  ARROW_LEFT_24
ARROW_UP_24
BOX_16
CAMERA_16
CAMERA_24
CHECK_CIRCLE_16
CHECK_CIRCLE_FILL_100
CHEVRON_DOWN_16
CHEVRON_LEFT_16
CHEVRON_LEFT_40
CHEVRON_RIGHT_14
CHEVRON_RIGHT_16
CHEVRON_RIGHT_40
CLOCK_16
CLOCK_24
CLOSE_16
CLOSE_24
FILTER_16
FILTER_24
GOOGLE_24
HEART_16
HEART_24
HEART_ACTIVE_16
HEART_ACTIVE_24
KAKAO_24
MEH_40
MEH_ACTIVE_24
MEH_ACTIVE_40
MENU_24
MESSAGE_100
MESSAGE_16
MESSAGE_24
MORE_16
MORE_24
PICTURE_40
REFRESH_16
REFRESH_24
SAD_40
SAD_ACTIVE_24
SAD_ACTIVE_40
SEARCH_24
SETTING_24
SMILE_16
SMILE_24
SMILE_40
SMILE_ACTIVE_24
SMILE_ACTIVE_40
STORE_16
STORE_24
TRIANGLE_DOWN_16
```
~~으어 효정.. 미쳤어 ㅠㅠㅠ 넘 고생해써..!!! 요거 asc로 정렬하는거 어떨까!?~~

아이고 정렬이 되어있었구나!! 착각 했어!!